<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_widget</class>
 <widget class="QWidget" name="chat_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>449</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>4</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string> [LoEWCT] Control Panel v0.5 LEGACY</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/winIcons/announcement.ico</normaloff>:/winIcons/announcement.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Chat Box - Simple Console</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="chatLog">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>441</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Celestia Redux</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Player List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="playerList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>351</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>40</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/btnIcons/icons/wrench_orange.png</normaloff>:/btnIcons/icons/wrench_orange.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>10</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>[LoEWCT] v0.6.0 | mlkj's Private Server v0.6.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Manual List Refresh</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/btnIcons/icons/arrow_refresh.png</normaloff>:/btnIcons/icons/arrow_refresh.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>20</width>
     <height>431</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/btnIcons/icons/server.png</normaloff>:/btnIcons/icons/server.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Server Controls</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Server</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/btnIcons/icons/lightning.png</normaloff>:/btnIcons/icons/lightning.png</iconset>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart Server</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/btnIcons/icons/arrow_refresh_small.png</normaloff>:/btnIcons/icons/arrow_refresh_small.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Server</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/btnIcons/icons/cancel.png</normaloff>:/btnIcons/icons/cancel.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="chatClrBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Chat Log</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/btnIcons/icons/bin.png</normaloff>:/btnIcons/icons/bin.png</iconset>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/btnIcons/icons/group.png</normaloff>:/btnIcons/icons/group.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Player Controls</string>
    </attribute>
    <widget class="QPushButton" name="kickPlyrBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Kick Player</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="kickReason">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kick Reason...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set a Player first by double clicking a player on
the list before doing any of these actions.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="sendBoxMsgBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send Message</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="msgBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Message Box message...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getPosBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getRotBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Rotation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getPDBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Get ponyData</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="plyrInfoBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Player Information</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="mapSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Map Select...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ponyville</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>canterlot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cloudsdale</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sweetappleacres</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gemmines</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ponyville library 1st floor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ponyville library 2nd floor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sugarcubecorner</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>raritysboutique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>everfree3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>everfree1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>zecoras</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>appaloosa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cottage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TM01</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="moveToMapBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Map Teleport</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pID1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>playerID 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pID2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>playerID 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tpPlyrBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Player Teleport</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="posXbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="posYbox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="posZbox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>z</string>
     </property>
    </widget>
    <widget class="QPushButton" name="posTpBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>230</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Position Teleport</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="banreason">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Ban Reason...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="banPlrBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ban Player</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PromoModBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Promote To Moderator</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="undoModBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>De-Mod</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="chatCmdLine">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendChatCmdLine">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>420</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbRefreshInterval">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>420</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>ms</string>
   </property>
   <property name="prefix">
    <string>Auto-Refresh Interval: </string>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
   <property name="value">
    <number>5000</number>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>chatLog</zorder>
  <zorder>label_2</zorder>
  <zorder>playerList</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>tabWidget</zorder>
  <zorder>chatCmdLine</zorder>
  <zorder>sendChatCmdLine</zorder>
  <zorder>spbRefreshInterval</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chatCmdLine</sender>
   <signal>returnPressed()</signal>
   <receiver>sendChatCmdLine</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
